### PR TITLE
feat:added categories

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
User can filter according to categories in the explore page
Updated Toast messages

### CODE REVIEW

File: src/pages/Explore/Explore.jsx [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
+					<span
+						className={`chip ${selectedCategory ? "" : "chip-active"}`}
+						onClick={() => dispatchCall("")}
+					>
+						All
+					</span>
+					{categories.map((category) => (
+						<span
+							className={`chip ${
+								selectedCategory === category.categoryName ? "chip-active" : ""
```
Comment (MiheerTamkhane) [inline src/pages/Explore/Explore.jsx:35]: try selectedCategory === category.categoryName && "chip-active". 
It will work same and look clean.

**Review by MiheerTamkhane — APPROVED** [review]
Code is good and readable, alos good work app is working fine and smooth

File: src/components/AsideNav/AsideNav.jsx [hunk 2]
```diff
@@ -7,28 +7,28 @@ export const AsideNav = () => {
 		<aside class="aside-nav">
```
Comment (OmKakatkar) [inline src/components/AsideNav/AsideNav.jsx:7]: Use `className` instead of `class`

**Review by OmKakatkar — APPROVED** [review]
Your code looks good to merge after doing a some correction, I left in a comment.

I really liked:
* Reducers and hooks have their own file and folders.
* The code is modular and easy to understand.

**Pull request merged**